pkg/kubectl/cmd/rollout/rollout_resume.go

```diff
@@ -38,7 +38,7 @@ import (
 type ResumeConfig struct {
 	resource.FilenameOptions
 
-	Resumer func(object *resource.Info) (bool, error)
+	Resumer func(object *resource.Info) ([]byte, error)
```

Comment:
Hi @kargakis thanks for coming back to this PR. This was asked for here: https://github.com/kubernetes/kubernetes/pull/28949#r72980078
this needs to change as it is passed into the CalculatePatch() function.